Formula/arkade.rb

```diff
@@ -32,9 +32,9 @@ def install
 
     bin.install_symlink "arkade" => "ark"
 
-    (zsh_completion/"_arkade").write Utils.safe_popen_read(bin/"arkade", "completion", "zsh")
-    (bash_completion/"arkade").write Utils.safe_popen_read(bin/"arkade", "completion", "bash")
-    (fish_completion/"arkade.fish").write Utils.safe_popen_read(bin/"arkade", "completion", "fish")
+    
+    
+    generate_completions_from_executable
```

Comment:
I feel like something like:

```suggestion
    generate_completions_from_executable("arkade")
```

or 

```suggestion
    generate_completions_from_executable("arkade", "completion")
```

would be more in keeping with our existing DSLs. It should be more obvious what command is getting run, I think.